docs/notes/2.22.x.md

```diff
@@ -10,6 +10,14 @@ We offer [formal sponsorship tiers for companies](https://www.pantsbuild.org/spo
 
 ## What's New
 
+### New options system 
```

Comment:
Minor, but can we put this under the `Highlights`? And maybe call it out as a highlight?

Reply:
What do you mean by call it out as a highlight?

Reply:
Oh, I see what you mean.